src/OpenTelemetry.Exporter.Geneva/CHANGELOG.md

```diff
@@ -1,6 +1,9 @@
-# Changelog
+# Changelog - OpenTelemetry.Exporter.Geneva
 
-## Unreleased
+## 0.0.1 [2022-Apr-19]
+
+This is teh first release for the `OpenTelemetry.Exporter.Geneva`
```

Comment:
```suggestion
This is the first release for the `OpenTelemetry.Exporter.Geneva`
```

Reply:
Or maybe 

```suggestion
This is a very first test release for the `OpenTelemetry.Exporter.Geneva`
```

Reply:
I think we could use "This is the first test release for ..."